test/re_test.cc

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 #include <gtest/gtest.h>
-#include "re.h"
+#include "../src/re.h"
```

Comment:
agreed - please add a TODO (or create an issue - or both!) to remove the re_tests and add the regex matching as a benchmark test against the public API.